plover/assets/main.json

```diff
@@ -74264,7 +74292,7 @@
 "PWOZ/TPHAOE/KWRA": "Bosnia",
 "PWOZ/TPHEU/KWRA": "Bosnia",
 "PWR": "<br>",
-"PWR*BGS": "{[}",
+"PWR*BGS": "{^[^}",
```

Comment:
Which stroke pairs with this one?
I see the following pairs:
```
"PWR-BT": "[",
"PWR*BT": "]",

"PWR-BGT": "{[^}",
"PWR*BGT": "{^]}",

"PWR-BG": "{^[^}",
"PWR-BGS": "{^]^}",

"PWR*BGS": "{^[^}",
```
So the last pair, and the last (this) stroke is a bit of a strange pair/triple compared to the first two pairs.
I don't see how changing it to `{[^}` would make sense though.